config/language_server_node_cdx.yml

```diff
@@ -1,9 +1,9 @@
 # TODO License.
 
-# Configuration for the JAVA LSP server component
+# Configuration for the CDS LSP server component
 ---
-version: 0.0.+
-repository_root: "https://buildpackcomponentrepositorycdx.cfapps.sap.hana.ondemand.com"
+version: 1.*.*
+repository_root: "https://lsp-component-cds.cfapps.eu10.hana.ondemand.com/"
 env:
     workdir: language_server_node_cdx/

```

Comment:
Change section below - we are now using stdio instead of sockets:
    ipc:
        protocol: "stream"

Reply:
done